tf2onnx/onnx_opset/nn.py

```diff
@@ -1117,7 +1145,11 @@ def version_7(cls, ctx, node, **kwargs):
         loop_node = ctx.make_node(op_type="Loop", inputs=[trip_cnt, cond.output[0], col_init],
                                   output_count=2, branches=branches)
         # convert generated mask matrix from bool to right shape and data type
-        squeeze = ctx.make_node(op_type="Squeeze", inputs=[loop_node.output[1]], attr={"axes": [squeeze_axis]})
```

Comment:
Can you remove this if/else and use graphbuilder in all cases?

Reply:
fixed